src/main/java/com/google/api/generator/gapic/composer/grpc/GrpcServiceStubClassComposer.java

```diff
@@ -289,7 +287,7 @@ private AnonymousClassExpr createRequestParamsExtractorAnonClass(Method method)
         VariableExpr.withVariable(
             Variable.builder().setType(method.inputType()).setName("request").build());
 
-    for (String httpBindingFieldName : method.httpBindings()) {
+    for (String httpBindingFieldName : method.httpBindings().pathParameters()) {
```

Comment:
Did you mean to nest this in `if (method.hasHttpBindings())`?

Reply:
No, this would be redundant, as it is already checked at the beginning of this method on line 258. We would get `IllegalStateException` much earlier than reaching this line.

Reply:
Thanks, this wasn't showing in the diff.